.github/ISSUE_TEMPLATE/adopter----add-company-name-.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Adopter -[ ADD COMPANY NAME]
+about: Help us track adoption so we can continue to improve our project
+title: ''
+labels: community
+assignees: DavidPHirsch
+
+---
+
+**Company details**
+Please add a link to your company website.
+
+**Adoption**
+Please list which components of OpenFeature you are currently adopting
```

Comment:
Should we add a question of details about if they are using it / intending to use it in prod?